README.md

```diff
@@ -271,6 +273,8 @@ If you're not inclined to make PRs you can tweet me at ```@ripienaar```
  * https://education.github.com/pack - As long as you're a student at a recognized university
 
 ## Docker Related
+### Official docker repository
+  * http://hub.docker.com/ - 1 private and unlimited public containers
```

Comment:
lets just combine these two into one cat called 'image hosting' or similar? rather than add a catagory that will only ever have one entry which seems pointless